src/Parser.php

```diff
@@ -86,7 +86,7 @@ private function buildBirthDateObject(string $full_number, int $year) {
       $this->data_components['gender'] = 'M';
     }
 
-    if (date('L', mktime(0, 0, 0, 1, 1, $year)) == 1) {
+    if (date('L', mktime(0, 0, 0, 1, 1, $year)) === "1")
```

Comment:
```suggestion
    if (date('L', mktime(0, 0, 0, 1, 1, $year)) === "1") {
```

There is a missing curly brace at the end, which prevents the tests from being run.